src/polyswarm_api/http.py

```diff
@@ -0,0 +1,57 @@
+from requests_futures.sessions import FuturesSession
+from concurrent.futures import ThreadPoolExecutor
+import requests
+from urllib3.util.retry import Retry
+from requests.adapters import HTTPAdapter
+
+from . import const
+
+
+class PolyswarmHTTPBase(object):
```

Comment:
I really like that we are abstracting some of the HTTP calls here. I think this could be part of each executor module, as different executors can use different http libs and strategies to run these things. We could provide some generic HTTP functions in a utils package, that would be something like `build_polyswarm_headers`, even create a `class` that renders it as a `dict` maybe. This could be used across multiple executors in order to have the headers always built in a consistent way.